setup.py

```diff
@@ -82,6 +86,13 @@ def finalize_options(self):
         pass
 
     def run(self):
+        pass
```

Comment:
I guess using an ABC `abstractmethod` would be too much?

Reply:
Definitely too much. In almost all situations 😉 
I'm not going to do this one unless you push me to. Is that OK?

Reply:
Just a passing comment, I really didn't expect you to make that change 👍